cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -1812,12 +1812,14 @@ if(ARROW_WITH_PROTOBUF)
   else()
     set(ARROW_PROTOBUF_REQUIRED_VERSION "2.6.1")
   endif()
-  if(ARROW_FLIGHT)
-    set(ARROW_PROTOBUF_ARROW_CMAKE_PACKAGE_NAME "ArrowFlight")
-    set(ARROW_PROTOBUF_ARROW_PC_PACKAGE_NAME "arrow-flight")
-  else()
+  if(ARROW_ORC
+     OR ARROW_HDFS
```

Comment:
Hmm, do we need `ARROW_HDFS`? It seems that HDFS related codes don't use ProtoBuf.

Reply:
Right, HDFS is not needed, but ORC and opentelemetry are.